### PR TITLE
Expose MachineName in mscorlib.

### PR DESCRIPTION
**Pull request by Priya91**
cc @jkotas @joshfree

### CODE REVIEW

**Comment by jkotas** [comment]
Could you please verify that the CoreCLR implementation that you are enabling here returns the same result as your proposed .NET Native / CoreRT implementation, for both UWP and non-UWP cases?

The behavior needs to be consistent. By looking at the code, there is likely difference for UWP at least - your proposed .NET Native / CoreRT implementation will return null for UWP, but CoreCLR will return the actual name for UWP. Perhaps, the CoreCLR implementation needs to be special-cased to return null (or even better - throw platform not supported exception) for UWP?

**Comment by Priya91** [comment]
@jkotas As discussed offline, i have updated the commit with the desired behavior. Thanks!

**Comment by ellismg** [comment]
Do you know if this is going to break mscorlib.asmmeta generation for the internal Desktop or CoreCLR builds?

**Comment by jkotas** [comment]
Yes, it will need mscorlib.asmmeta update once it is mirrored to TFS...

**Comment by jkotas** [comment]
LGTM

File: src/mscorlib/src/System/Environment.cs [hunk 1]
```diff
@@ -1239,9 +1249,9 @@ public static IDictionary GetEnvironmentVariables()
             }
         }
 
-        #if FEATURE_CORECLR
+#if FEATURE_CORECLR
```
Comment (stephentoub) [inline src/mscorlib/src/System/Environment.cs]: Nit: looks like there are a bunch of unintentional formatting changes in the file

**Comment by Priya91** [comment]
Thanks for the review, will keep in mind to update mscorlib.asmmeta internally.

File: src/mscorlib/src/System/Environment.cs (hunk 1 above)
Comment (Priya91) [inline src/mscorlib/src/System/Environment.cs]: This is the right formatting I believe, VS auto formatted when updating the #if preprocessor statements.

**Pull request merged**